stacks/cert-manager-stack.yml

```diff
@@ -0,0 +1,21 @@
+services:
+  cert-manager:
+    global_permissions:
+    - '* certmanager.k8s.io/*'
+    - '* extensions/ingresses'
+    - '* configmaps'
+    - '* secrets'
+    - '* events'
+    - '* services'
+    - '* pods'
+    image: "daishan1992/cert-manager:latest"
```

Comment:
Can you add a setting for this and tokenize in the stack file `${IMAGE}` and pass it in as an answer?